src/Sarif.WorkItems/WorkItemsResources.resx

```diff
@@ -119,7 +119,7 @@
   </resheader>
   <data name="AzureDevOpsDefaultDescriptionFooter" xml:space="preserve">
     <value>&lt;br&gt;&lt;br&gt;For another viewing option, load the attached SARIF log file in: &lt;br&gt;
-&lt;li&gt;The &lt;a href="https://marketplace.visualstudio.com/_apis/public/gallery/publishers/WDGIS/vsextensions/MicrosoftSarifViewer/2.1.7/vspackage"&gt;Visual Studio SARIF add-in&lt;/a&gt;.&lt;/li&gt;
+&lt;li&gt;The &lt;a href="https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer"&gt;Visual Studio SARIF add-in&lt;/a&gt;.&lt;/li&gt;
```

Comment:
Today, we have two extensions:
- one for old VS versions
- one for VS2022
- one for VSCode

With that in mind, I think we should update this message and show the other options.

Reply:
Yes makes sense, so just to confirm we should ideally display these 4 options as below:
1) https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer (2017, 2019)
2) https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer2022 (2022)
3) https://marketplace.visualstudio.com/items?itemName=MS-SarifVSCode.sarif-viewer (VSCode)
4) https://microsoft.github.io/sarif-web-component/ (Web)

I can make these changes once you confirm. Thanks Eddy!

Reply:
Sounds perfect.
Once you do, can you add a screenshot showing the message and where it would appear, pls?

Reply:
Great, thanks Eddy! I will modify the PR for this and it should look something like this:

![image](https://user-images.githubusercontent.com/42941591/162545140-bb256165-736d-456d-b914-42b3118afa01.png)